repositories.bzl

```diff
@@ -62,6 +62,10 @@ DIST_ARCHIVE_REPOS = [get_canonical_repo_name(repo) for repo in [
     "rules_foreign_cc+",
     "rules_fuzzing+",
     "rules_swift+",
+    "grpc-java++grpc_java_repositories_extension+com_github_cncf_xds",
+    "grpc-java++grpc_java_repositories_extension+envoy_api",
+    "grpc-proto+",
+    "grpc++grpc_repo_deps_ext+opencensus_proto",
```

Comment:
could you move this line to after line 58? no particular reason, just the way it's currently arranged.